src/constants/wallets.ts

```diff
@@ -136,161 +58,51 @@ type WalletConfig = {
-    matchesInjectedEip1193: (provider) => provider.isHbWallet,
-  },
-  [WalletType.ImToken]: {
-    type: WalletType.ImToken,
-    stringKey: STRING_KEYS.IMTOKEN,
-    icon: ImTokenIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) => provider.isImToken,
-    walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.imToken,
-  },
-  [WalletType.MathWallet]: {
-    type: WalletType.MathWallet,
-    stringKey: STRING_KEYS.MATH_WALLET,
-    icon: MathWalletIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193],
-    matchesInjectedEip1193: (provider) => provider.isMathWallet,
-    walletconnect2Id: '7674bb4e353bf52886768a3ddc2a4562ce2f4191c80831291218ebd90f5f5e26',
-  },
-  [WalletType.MetaMask]: {
-    type: WalletType.MetaMask,
-    stringKey: STRING_KEYS.METAMASK,
-    icon: MetaMaskIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) =>
-      Boolean(provider.isMetaMask) && !provider.overrideIsMetaMask,
-    walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.Metamask,
   },
   [WalletType.OkxWallet]: {
     type: WalletType.OkxWallet,
     stringKey: STRING_KEYS.OKX_WALLET,
     icon: OkxWalletIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) => provider.isOkxWallet,
     walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.OkxWallet,
   },
-  [WalletType.Rainbow]: {
-    type: WalletType.Rainbow,
-    stringKey: STRING_KEYS.RAINBOW_WALLET,
-    icon: RainbowIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) => provider.isRainbowWallet,
-    walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.Rainbow,
-  },
-  [WalletType.TokenPocket]: {
-    type: WalletType.TokenPocket,
-    stringKey: STRING_KEYS.TOKEN_POCKET,
-    icon: TokenPocketIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) => provider.isTokenPocket,
-    walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.TokenPocket,
-  },
-  [WalletType.TrustWallet]: {
-    type: WalletType.TrustWallet,
-    stringKey: STRING_KEYS.TRUST_WALLET,
-    icon: TrustWalletIcon,
-    connectionTypes: [WalletConnectionType.InjectedEip1193, WalletConnectionType.WalletConnect2],
-    matchesInjectedEip1193: (provider) => provider.isTrust,
-    walletconnect2Id: WALLET_CONNECT_EXPLORER_RECOMMENDED_WALLETS.Trust,
-  },
-  [WalletType.WalletConnect2]: {
-    type: WalletType.WalletConnect2,
-    stringKey: STRING_KEYS.WALLET_CONNECT_2,
-    icon: WalletConnectIcon,
-    connectionTypes: [WalletConnectionType.WalletConnect2],
-  },
   [WalletType.Keplr]: {
```

Comment:
Is this actually supporting Keplr EVM, which is interesting (I didn't even know Keplr had evm). I don't think we were planning to surface the Keplr EVM button though.

<img width="501" alt="Screenshot 2024-08-28 at 2 07 49 PM" src="https://github.com/user-attachments/assets/fba19921-69e9-40af-b5ad-2c180007bdb6">

Reply:
filtered out keplr evm!! thanks for catching!